app/transformers/api/v4/teleconsultation_transformer.rb

```diff
@@ -0,0 +1,20 @@
+class Api::V4::TeleconsultationTransformer
+  class << self
+    def to_response(teleconsultation)
+      Api::V4::Transformer.to_response(teleconsultation)
+        .except(*Teleconsultation::REQUEST_ATTRIBUTES)
+        .except(*Teleconsultation::RECORD_ATTRIBUTES)
+        .merge({"request" => teleconsultation.request,
+                "record" => teleconsultation.record})
+    end
+
+    def from_request(teleconsultation)
+      request, record = teleconsultation["request"], teleconsultation["record"]
+      payload = Api::V4::Transformer.from_request(teleconsultation).except("request", "record")
+
+      payload.merge!(request) if request
```

Comment:
What do we think about making the data model change right away and update the PD's `teleconsultation_id` in a `MergeTeleconsultationService` over storing the PD IDs in an array on `Teleconsultation`?

It'd make the migration easier later, since you'll only have to change code-paths and not migrate any data. We can `except` that `teleconsultation_id` from the `prescription_drug` payload while syncing until the app can understand it.

Reply:
The annoying part is, there's no guarantee the PD will be synced when this request is received. That's part of the reason why we decided to bite the bullet and strip out these arrays everywhere - [ref](https://simpledotorg.slack.com/archives/CV9V1DWLV/p1600251647036000)